contents/copy/cfs/intro.md

```diff
@@ -0,0 +1,6 @@
+---
+title: Call For Speakers
+introText: >
+  CSSconf EU is targeted at CSS professionals, CSS creatives, CSS aficionados. It doesn’t matter whether you are a professional front end developer, a designer who practices CSS magic in their spare time, or you just love CSS and want to share your knowledge. We’re interested in topics that make us go, "Oh, wow! Yes! Cool! I want to hear about that!"
```

Comment:
"CSS professionals, creatives, and aficionados"

Reply:
can you edit and fix?